generate_schedule.py

```diff
@@ -426,23 +426,23 @@ def generate_schedule(args):
     # (3) positive loads => discharge
     # (4) percentile with highest load => discharge
     # Note: The procedure to determine priorities for every timestep assumes that
-    # time intervals of simulation are equal to time intervals in NSM time series.
+    # time intervals of simulation are equal to time intervals in grid situation time series.
 
-    # compute cutoff values for priorities 1 and 4 using all netto values
-    idx_percentile = int(len(netto) * args.priority_percentile)
-    sorted_netto = sorted(netto)
-    cutoff_priority_1 = sorted_netto[idx_percentile]
-    cutoff_priority_4 = sorted_netto[len(netto) - idx_percentile]
+    # compute cutoff values for priorities 1 and 4 using all residual load values
+    idx_percentile = int(len(residual_load) * args.priority_percentile)
+    sorted_residual_load = sorted(residual_load)
+    cutoff_priority_1 = sorted_residual_load[idx_percentile]
+    cutoff_priority_4 = sorted_residual_load[len(residual_load) - idx_percentile]
 
     # find timesteps relevant for simulation and discard remaining
-    idx_start = (s.start_time.replace(tzinfo=None) - nsm_start_time) // s.interval
-    idx_start = idx_start if 0 < idx_start < len(netto) else 0
-    idx_end = min(idx_start + s.n_intervals, len(netto))
-    netto = netto[idx_start:idx_end]
+    idx_start = (s.start_time.replace(tzinfo=None) - grid_start_time) // s.interval
+    idx_start = idx_start if 0 < idx_start < len(residual_load) else 0
+    idx_end = min(idx_start + s.n_intervals, len(residual_load))
+    residual_load = residual_load[idx_start:idx_end]
     curtailment = curtailment[idx_start:idx_end]
 
     # zero-pad for same length as scenario
-    netto += [0]*(s.n_intervals - len(netto))
+    residual_load += [0] * (s.n_intervals - len(residual_load))
     curtailment += [0]*(s.n_intervals - len(curtailment))
```

Comment:
should be same style (spaces around operators or not). Spaces seem to be more common, so please change lines 446, 449 and 470 as well.